pkg/types/alibabacloud/platform.go

```diff
@@ -7,11 +7,32 @@ type Platform struct {
 
 	// ResourceGroupID is the ID of an already existing resource group where the cluster should be installed.
 	// This resource group must be empty with no other resources when trying to use it for creating a cluster.
-	// If empty, a new resource group will created for the cluster.
-	// Destroying the cluster using installer will delete this resource group.
+	// If empty, a new resource group will created for the cluster,
+	// and destroying the cluster using installer will delete this resource group.
 	// +optional
 	ResourceGroupID string `json:"resourceGroupID,omitempty"`
 
+	// VpcID is the ID of an already existing VPC where the cluster should be installed.
+	// If empty, a new VPC will created for the cluster, and destroying
```

Comment:
I do not think that the text about destroying the VPC should be added. If the installer creates the VPC, it is contracted that the installer will delete the VPC. If the installer does not delete the VPC, then that needs to be called out explicitly. If the installer is deleting the user-supplied VPC, then that needs to be called out explicitly (as was done for the resource group). Otherwise, nothing needs to be stated.